bitnami/postgresql/Chart.yaml

```diff
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 12.11.3
+version: 12.11.4
```

Comment:
After rebasing origin/main, can you please bump the version again?

If the conflict happens again before we finally merge the PR, we will do the bump ourselves.